server/models/queryHistory.js

```diff
@@ -0,0 +1,68 @@
+const db = require('../lib/db.js');
+const Joi = require('@hapi/joi');
+const config = require('../lib/config');
+
+const schema = Joi.object({
```

Comment:
Can we also capture queryId if it exists? I think a nice enhancement following this is to have the history for a specific query available in case user wants to restore a previous version (though maybe that might get weird with partial query executions when highlighting a snippet of SQL?)

Reply:
that makes sense, I will add it

Reply:
added by 6df0666